CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contribution guidelines
+
+If you want to contribute, we ask you to follow these guidelines.
+
+## Reporting bugs
+
+If you have encountered a bug in this project, please check if an issue already 
+exists in the list of existing [issues][issues]. If such an issue does not 
+exist, you can create a [new issue][new_issue]. When writing the bug report, 
+try to add a clear example that shows how to reproduce said bug.
+
+## Adding new features
+
+Before making making changes to the code, we advise you to first check the list 
+of existing [issues][issues] for this project to see if an issue for the 
+suggested changes already exists. If such an issue does not exist, you can 
+create a [new issue][new_issue]. Creating an issue gives an opportunity for 
+other developers to give tips even before you start coding.
+
+### Code style
+
+To keep the code clean and readable, this project uses:
+
+- [`isort`](https://github.com/timothycrosley/isort) to order the imports
+- [`black`](https://github.com/psf/black) to format the code and keep diffs for 
+  pull requests small
+- [`flake8`](https://github.com/PyCQA/flake8) to clean up code (removing unused
+  imports, etc.)
```

Comment:
These tools are of course not relevant in a client-side project :-)

However, what applies is the linting done by `ESLint` as part of the create-react-app stack. Developers don't have to do anything for this and CI will catch the mistakes - only thing is that they need to check the console/build output.